synapse/axon.py

```diff
@@ -593,7 +593,8 @@ async def dels(self, sha256s):
         await self._reqUserAllowed(('axon', 'del'))
         return await self.cell.dels(sha256s)
 
-    async def wget(self, url, params=None, headers=None, json=None, body=None, method='GET', ssl=True, timeout=None, proxy=None):
+    async def wget(self, url, params=None, headers=None, json=None, body=None, method='GET',
+                   ssl=True, timeout=None, proxy=None, ssl_opts=None):
```

Comment:
Do we need to add a version flag to the axon cellinfo to note the ssl_opts support?

Reply:
Per discussion added version checking